### DISCUSSION TITLE
Stop tablets

### DISCUSSION THREAD

**Discussion by heinlyfamily2001** [GrandOrgue/GoOdf#56] [General]
Could it be possible to add the "tongue style" stop tabs to the software?

**Comment by larspalo** [comment]
GOODF is just a software to create odfs for GrandOrgue, in itself it doesn't provide the images (but you as a user can). A request for a new style of drawstops in the form of stop tabs must be submitted to be included in GrandOrgue instead first.

Any new style of drawstops (tabs) must comply with the currently existing ones to be included in GO. This means that the new bitmaps should:

- Have a resolution of exactly 65 x 65 pixels
- Have both on and off states that completely exchange the whole image area (ie. provide its own background covering both states)
- Be in a .png format
- Be licensed as GNU GPL v2 (or later)

Feel free to create such images and suggest them for inclusion in GrandOrgue as a PR.
